masonry/src/widgets/flex.rs

```diff
@@ -801,32 +775,30 @@ impl Widget for Flex {
                     let child_pos = border.place_down(child_pos);
                     let child_pos = padding.place_down(child_pos);
                     ctx.place_child(widget, child_pos);
+
                     major += self.direction.major(child_size);
                     major += space_between;
-                    major += gap;
+                    major += self.gap;
                 }
                 Child::FlexedSpacer(_, calculated_size)
                 | Child::FixedSpacer(_, calculated_size) => {
                     major += *calculated_size;
-                    major += gap;
+                    major += self.gap;
                 }
             }
         }
 
-        if flex_sum > MIN_FLEX_SUM && total_major.is_infinite() {
+        if flex_sum > MIN_FLEX_SUM && bc_major.is_infinite() {
             tracing::warn!("A child of Flex is flex, but Flex is unbounded.");
         }
 
         let final_major = if flex_sum > MIN_FLEX_SUM || self.fill_major_axis {
-            total_major.max(major_non_flex)
+            bc_major.max(major_non_flex)
         } else {
             major_non_flex
         };
 
-        // my_size may be larger than the given constraints.
-        // In which case, the Flex widget will either overflow its parent
-        // or be clipped (e.g. if its parent is a Portal).
```

Comment:
This comment seems fairly redundant, given that the same applies to any widget.

We should have a whole documentation chapter about overflow, but in the meantime, this paragraph doesn't really give any useful info.

Reply:
It was a good place for the comment. Hopefully we can get the documentation chapter done soon.